Source/Core/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ add_subdirectory(VideoBackends)
 
 if(ENABLE_NOGUI)
   add_subdirectory(DolphinNoGUI)
+  add_subdirectory(DolphinTool)
```

Comment:
This seems odd to me, why is this only built if `ENABLE_NOGUI` is set?

Reply:
Would one need it outside that context?

Reply:
The nogui version is fairly niche, with the main use I can think of being fifoci.  It's not built for windows by default and not included in the updater or on the download page.  I could see wanting to build the CLI tool but not the nogui version (and probably, the CLI tool should be distributed for windows, especially since DSPTool currently is).

I suspect part of the reason why the nogui version isn't included by default is that it's ~10 MB, with the main dolphin EXE being ~14 MB (not counting the Qt DLLs); DSPTool is pretty small (~270 KB), and if the CLI tool is small there wouldn't be much of a reason to exclude it.

Reply:
Should it maybe get its own flag e.g. ENABLE_TOOL?

Reply:
Yeah, I think that a separate flag would be good (though perhaps `ENABLE_CLI_TOOL` would be a clearer name?)